net/smartdns/Makefile

```diff
@@ -0,0 +1,79 @@
+
+# Copyright (C) 2018-2020 Ruilin Peng (Nick) <pymumu@gmail.com>.
+#
+# smartdns is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# smartdns is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=smartdns
+
+PKG_SOURCE_PROTO:=git
```

Comment:
Try to match the template here: https://github.com/openwrt/packages/blob/master/net/cifsd-tools/Makefile#L6

Reply:
why that one?  this is completely arbitrary.  Why is this one any _worse_ ?

Reply:
the newewst smatdns release tag is 28, should I change it back?